src/containers/shared/components/Transaction/Clawback/Description.tsx

```diff
@@ -7,7 +7,10 @@ import { formatAmount } from '../../../../../rippled/lib/txSummary/formatAmount'
 export const Description = ({ data }: TransactionDescriptionProps) => {
   const issuer = data.tx.Account
   const holder = data.tx.Amount.issuer
-  const amount = data.tx.Amount
+
+  // construct a new amount object by spreading to avoid mutating
+  // the original object reference
+  const amount = { ...data.tx.Amount }
```

Comment:
Is this an issue in the parser too?

Reply:
i dont believe so. In the parser, it calls `formatAmount`, which returns a new object rather an reference to the raw data

Reply:
Why not call `formatAmount` in this too?

Reply:
that works too. will make change